health/notifications/alerta/metadata.yaml

```diff
@@ -0,0 +1,93 @@
+# yamllint disable rule:line-length
+---
+- id: 'notify-alerta'
+  meta:
+    name: 'Alerta'
+    link: 'https://alerta.io/'
+    categories:
+      - notify.agent
+    icon_filename: 'alerta.png'
+  keywords:
+    - Alerta
+  overview:
+    notification_description: |
+      The [Alerta](https://alerta.io/) monitoring system is a tool used to consolidate and de-duplicate alerts from multiple sources for quick ‘at-a-glance’ visualization. With just one system you can monitor alerts from many other monitoring tools on a single screen.
+      You can send Netdata alerts to Alerta to see alerts coming from many Netdata hosts or also from a multi-host Netdata configuration.
+    notification_limitations: ''
+  setup:
+    prerequisites:
+      list:
+        - title: ''
+          description: |
+            - A working Alerta instance
+            - An Alerta API key (if authentication in Alerta is enabled)
+            - Access to the terminal where Netdata Agent is running
+    configuration:
+      file:
+        name: 'health_alarm_notify.conf'
```

Comment:
Should there be a path prefix of sorts here to be clear where it lives?

Reply:
For files such as those, we e.g. for python.d stuff we mention `python.d/module.conf`. The `/etc/netdata/` is variable, could be I think `/opt/` for static builds.

Since `health_alarm_notify.conf` lives right under `/etc/netdata` I think it should be ok just like this?